stride/problem/data.py

```diff
@@ -1604,13 +1604,25 @@ def plot(self, **kwargs):
         """
         title = kwargs.pop('title', self.name)
         plot = kwargs.pop('plot', True)
+        plot_type = kwargs.pop('plot_type', 'gather')
         if self.time.num == self.shape[1]:
             time_axis = self.time.grid / 1e-6
         else:
             time_axis = np.arange(self.shape[1])
 
-        axis = plotting.plot_gather(self.transducer_ids, time_axis, self.data,
-                                    title=title, **kwargs)
+        if plot_type == 'gather':
+            axis = plotting.plot_gather(self.transducer_ids, time_axis, self.data,
+                                        title=title, **kwargs)
+        elif plot_type == 'spectrum':
+            sampling_rate = 1/self.time.step
+            axis = plotting.plot_magnitude_spectrum(sampling_rate, self.transducer_ids, time_axis, self.data,
+                                                    title=title, **kwargs)
+        elif plot_type == 'spectrum_per_trace':
```

Comment:
I don't quite like the names. Per trace is not very descriptive. Can we think of some other alternative?